pkg/supportbundle/supportbundle.go

```diff
@@ -57,9 +58,22 @@ func CollectSupportBundleFromSpec(spec *troubleshootv1beta2.SupportBundleSpec, a
 	}
 	defer os.RemoveAll(tmpDir)
 
-	basename := fmt.Sprintf("support-bundle-%s", time.Now().Format("2006-01-02T15_04_05"))
-	if !opts.FromCLI {
-		basename = filepath.Join(os.TempDir(), basename)
+	basename := ""
+	if opts.OutputPath != "" {
+		// use override output path
+		overridePath, err := filepath.Abs(opts.OutputPath)
+		if err != nil {
+			return nil, errors.Wrap(err, "")
+		}
+
+		basename = filepath.Base(overridePath)
```

Comment:
this looks like it strips off the path and will always write to the current directory on line 129

https://github.com/replicatedhq/troubleshoot/pull/532/files#diff-04328572fa60f5b4cdaaa40835e6d53387d06e87f3e4becf95ab471678323bb6R129